client/src/components/Swap/SwapModal.tsx

```diff
@@ -179,6 +206,8 @@ export const SwapModal: React.FC<SwapModalProps> = ({
             onClick={async () => {
               handleCompleteClick();
             }}
+            bgColor={ isMobile ? (paymentInitiated ? '#df2e2d' : '#ffcccc') : '#df2e2d'
```

Comment:
Recommend using disabled state instead. Also add color constants to theme/colors

Reply:
I am in a split on how to do this. Feel like the user needs to be shown only one CTA on the send payment page. But we can't just disable the other button until the pay button is clicked because the user might have paid and is coming back to this page again from somewhere else in the app, and hence shouldn't be forced to click on the pay button to enable the "I have completed payment". I think we need a more complex state management here which caches the first click in parent components rather than just storing it in state which will get reset on rendering.